nodejs/week3/src/backend/api/meals.js

```diff
@@ -0,0 +1,90 @@
+const express = require("express");
+const app = express();
```

Comment:
Seems to be unused

Reply:
Thanks!. Corrected in PR: https://github.com/Lucha2040/hyf-homework/pull/36/commits/586da540138991a0d004e8f01f601105b8d20d4d